src/Controller/SalesChannelApiController.php

```diff
@@ -27,10 +27,12 @@
 
 use Adyen\Shopware\Service\PaymentDetailsService;
 use Adyen\Shopware\Service\PaymentMethodsService;
+use Adyen\Shopware\Service\PaymentStatusService;
+use http\Env\Response;
```

Comment:
I don't think you will need this
```suggestion
```

Reply:
Indeed, generated by the IDE at some point